nuxeo-core/nuxeo-core-event/src/test/resources/test-stream-workmanager-service.xml

```diff
@@ -16,4 +16,8 @@
     <logConfig name="work" />
   </extension>
 
+  <extension target="org.nuxeo.runtime.ConfigurationService" point="configuration">
+    <property name="nuxeo.work.computation.store.state.boolean">true</property>
```

Comment:
Please add documentation stating what this parameter does and the since version.